resources/api-gateway-migrator/templates/rbac.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "api-gateway-migrator.name" . }}-account
+  namespace:  {{ .Release.Namespace }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "api-gateway-migrator.name" . }}-role
+  namespace:  {{ .Release.Namespace }}
+rules:
+    #API Controller resources
+  - apiGroups: ["apiextensions.k8s.io", "gateway.kyma-project.io", "extensions", "authentication.istio.io", "networking.istio.io"]
+    resources: ["customresourcedefinitions", "apis", "policies", "virtualservices"]
+    verbs: ["*"]
+  - apiGroups: [""]
+    resources: ["services", "namespaces"]
+    verbs: ["get", "list", "watch", "delete"]
+    #API Gateway resources
+  - apiGroups: ["gateway.kyma-project.io"]
+    resources: ["apirules", "apirules/status"]
+    verbs: ["*"]
+  - apiGroups: ["networking.istio.io"]
+    resources: ["virtualservices"]
```

Comment:
Do we really need those grants here? The job is meant to create apirules. VSs and Oathkeeper rules are handled by the apigateway controller.

Reply:
fixed